src/components/recommandations/RecoReport.tsx

```diff
@@ -9,14 +9,15 @@ import {
   Text,
 } from "@chakra-ui/react";
 import { css } from "@emotion/css";
-import React, { useEffect, useState } from "react";
-import { useDispatch } from "react-redux";
+import { isEqual } from "lodash";
+import React, { useCallback, useEffect, useState } from "react";
+import { useDispatch, useSelector } from "react-redux";
```

Comment:
```suggestion
import { useDispatch } from "react-redux";
```
useSelector does not seem to be used